resources/views/projects/my-projects.blade.php

```diff
@@ -79,18 +81,26 @@
         @endif
     @endif
 
-    @switch($user->context)
-        @case('organization')
-            @include(isset($section) ? 'projects.my-projects.' . $section : 'projects.my-projects.running')
-        @break
+    @if (Auth::user()->can('viewAny', 'App\Models\Project'))
+        @switch($user->context)
+            @case('organization')
+                @include(isset($section) ? 'projects.my-projects.' . $section : 'projects.my-projects.running')
+            @break
 
-        @case('regulated-organization')
-            @includeWhen($projectable, 'projects.my-projects.running')
-        @break
+            @case('regulated-organization')
+                @includeWhen($projectable, 'projects.my-projects.running')
+            @break
 
-        @default
-            @include(isset($section) ? 'projects.my-projects.' . $section : 'projects.my-projects.participating')
-    @endswitch
+            @default
+                @include(isset($section) ? 'projects.my-projects.' . $section : 'projects.my-projects.participating')
+        @endswitch
+    @else
+        @if ($user->context === 'regulated-organization')
+            @includeWhen($projectable, 'projects.my-projects.running')
+        @elseif ($user->context === 'organization')
+            @include('projects.my-projects.running')
+        @endif
```

Comment:
If I'm reading this correctly, for regulated-organizations you only include the `project.my-projects.running` partial when there is a `$projectable` but for organizations you always include it. Is that correct and if so why the distinction?

I also wonder if you could collapse this down into a single `@includeWhen` statement.

Reply:
Thanks @jobara for pointing this out, I just followed what was there before, and only regulated-organization was using includeWhen to show the running blade. @includeWhen seems to be working for organization account as well, so I will update this to use single @includeWhen.